### DISCUSSION TITLE
Updated Address Doesn't Go To New Address

### DISCUSSION THREAD

**Discussion by JTLovato** [emailjs-com/help#145] [Q&A]
Hello,
I recently signed up using one of my original emails, assisting someone else with their site. The plan was after completing it, I'd change the email and password and hand it off to them.
Unfortunately, even after changing the email and connecting their gmail account, the login is still my original email. I was able to update the password but not their email. I've updated the connected service under the email services and made sure their email is connected.
Any assistance is appreciated, thank you!

**Comment by xr0master** [comment]
To change your login address, contact support via the dashboard. Tell them your new address, and they will help you.
If you have a Professional plan or above, you can add any user yourself.
